docs/nodes/build/launch-an-avalanche-validator-on-aws-with-one-click.md

```diff
@@ -0,0 +1,347 @@
+
+![AMI instance](/img/one-click-validator-node/ami-instance.png)
+
+Copy that public IP address and open a Terminal or command line prompt. Once you
+have the new Terminal open `ssh` into the EC2 instance with the following
+command. 
+
+```zsh
+ssh username@ip.address.of.ec2.instance
+```
+
+Now that you are `ssh`ed into the EC2 instance you can run the
+`info.isBoostrapped` command to confirm if the Avalanche Validator node has
+finished bootstrapping.
+
+### `info.isBootstrapped` Request
+
+```zsh
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.isBootstrapped",
+    "params": {
+        "chain":"X"
+    }
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+Once the node is finished bootstrapping, the response will be:
+
+### `info.isBootstrapped` Response
+
+```zsh
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "isBootstrapped": true
+    },
+    "id": 1
+}
+```
+
+**Note** that initially the response is `false`. This is because it takes time
+for the node to sync the Avalanche Fuji testnet. When you are adding your node
+as a Validator on the Avalanche Mainnet you will want to wait for this response
+to return `true` so that you don't suffer from any downtime while validating.
+For this tutorial we're not going to wait for it to finish syncing as it's not
+strictly necessary.
+
+### `info.getNodeID` Request
+
+Next, you want to get the nodeID which will be used to add the node as a
+Validator. To get the node's ID you call the `info.getNodeID` jsonrpc endpoint.
+
+```zsh
+curl --location --request POST 'http://127.0.0.1:9650/ext/info' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.getNodeID",
+    "params" :{
+    }
+}'
+```
+
+### `info.getNodeID` Response
+
+Take a note of the `nodeID` value which is returned as you'll need to use it in
+the next step when adding a validator via the Avalanche Web Wallet. In this case
+the `nodeID` is `NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5`
+
+```zsh
+{
+  "jsonrpc": "2.0",
+  "result": {
+    "nodeID": "NodeID-Q8Gfaaio9FAqCmZVEXDq9bFvNPvDi7rt5",
+    "nodePOP": {
+      "publicKey": "0x85675db18b326a9585bfd43892b25b71bf01b18587dc5fac136dc5343a9e8892cd6c49b0615ce928d53ff5dc7fd8945d",
+      "proofOfPossession": "0x98a56f092830161243c1f1a613ad68a7f1fb25d2462ecf85065f22eaebb4e93a60e9e29649a32252392365d8f628b2571174f520331ee0063a94473f8db6888fc3a722be330d5c51e67d0d1075549cb55376e1f21d1b48f859ef807b978f65d9"
+    }
+  },
+  "id": 1
+}
+```
+
+## Add Node as Validator on Fuji via the Web Wallet
+
+For adding the new node as a Validator on the Mainnet's Primary Subnet we can
+use the [Avalanche Web Wallet](https://wallet.avax.network).
+
+![Avalanche Web Wallet](/img/one-click-validator-node/web-wallet.png)
+
+The Avalanche Web Wallet is a web-based application with no middleware or any
+kind of server communication. It's written in Vue JS and can be either accessed
```

Comment:
Probably don't need to mention the Vue JS part as it may change down the road.

Reply:
Resolved